diesel/src/mysql/connection/raw.rs

```diff
@@ -195,6 +196,7 @@ impl Drop for RawConnection {
 /// > any other client library call.
 ///
 /// <https://dev.mysql.com/doc/refman/5.7/en/mysql-init.html>
+#[allow(deprecated)] // min 1.38.0
```

Comment:
In my opinion, `Once::new()` has been stabilized in 1.2.0 so using it might not break compatibility...?

Reply:
When did it become a `const fn`, though? Otherwise we can't use it in a static AFAIK

Reply:
Ah, I might wrong but I think we can in static from 1.24.0, not 1.2.0 [godbolt](https://rust.godbolt.org/z/0hHkDO).

Reply:
Ah, very cool! Didn't know it's been available that long. Thanks for checking!